x/ibc/03-connection/keeper/keeper_test.go

```diff
@@ -0,0 +1,169 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	client "github.com/cosmos/cosmos-sdk/x/ibc/02-client"
+	clientexported "github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/03-connection/types"
+	commitment "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+	"github.com/stretchr/testify/suite"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	dbm "github.com/tendermint/tm-db"
+)
+
+const (
+	clientType = clientexported.Tendermint
+	storeKey   = "ibc"
+
+	ChainIDGaia1 = "gaia-1"
+	ChainIDGaia2 = "gaia-2"
```

Comment:
nit, can we rename the chain 2 to prevent confusion with the `gaia` variables defined on the tests ?

Reply:
Right.